Code/Editor/AzAssetBrowser/AzAssetBrowserWindow.cpp

```diff
@@ -98,6 +98,8 @@ AzAssetBrowserWindow::AzAssetBrowserWindow(QWidget* parent)
 
     m_ui->m_collapseAllButton->setAutoRaise(true); // hover highlight
     m_ui->m_collapseAllButton->setIcon(QIcon(AzAssetBrowser::CollapseAllIcon));
+    m_ui->m_collapseAllButton->setToolTip(tr("Collapse All"));
+    m_ui->m_collapseAllButton->setToolTipDuration(10000);
```

Comment:
Do we need this custom tooltip duration?

Reply:
Removed!